cmd/pipeline/destroy.go

```diff
@@ -124,7 +130,24 @@ func (pc *Command) ExecuteDestroyPipeline(ctx context.Context, opts *DestroyOpti
 	return nil
 }
 
-func (pc *Command) destroyPipeline(ctx context.Context, name, namespace string, destroyVolumes, destroyDependencies bool) (*types.GitDeployResponse, error) {
+func shouldDestroyDependencies(opts *DestroyOptions) bool {
+	isForceDestroyDependenciesSetInOktetoInstance := env.LoadBoolean(constants.OktetoForceDestroyDependencies)
+	isInsideDependency := env.LoadBoolean(constants.OktetoIsDependencyEnvVar)
+	if isInsideDependency {
+		return false
+	}
+	if !isInsideDependency && isForceDestroyDependenciesSetInOktetoInstance {
```

Comment:
```suggestion
	if isForceDestroyDependenciesSetInOktetoInstance {
```

Same here, this is not needed anymore after moving the other check, right? This can be simplified as the other function similar to this one